docker/release/Dockerfile

```diff
@@ -1,15 +1,22 @@
 FROM ubuntu:16.04
 
 RUN apt-get update; \
-    apt-get install -y libc-ares-dev
+    apt-get install -y libc-ares-dev jq
+
+COPY wait-for-it.sh run-iroha.sh go-grpc-health /
```

Comment:
Well, you can then build this binary yourself in ci
https://GitHub.com/warchant/go-grpc-gealthcheck

Reply:
Crosacompilation is super easy

Reply:
The problem is that it would break existing CI as it does not consider having alien architecture binaries in the Docker image. It should be modified accordingly to support compilation and including of this healthecking binary

Reply:
I will remove healthcheck and binary, because it is not possible to know at the docker image build time at what port iroha will be listening :(